src/components/Contentful/StoryCard.vue

```diff
@@ -101,11 +101,11 @@ export default {
 	.story-card__imageCard {
 		aspect-ratio: 4 / 5;
 		border-radius: 2.5rem;
-		min-height: 550px;
 		overflow: hidden;
 		padding: 2.5rem 1.25rem;
 		@include breakpoint(medium) {
 			padding: 2.5rem;
+			min-height: 550px;
```

Comment:
This is too tall for mobile devices causing the cards to overlap and overflow the viewport.

![image](https://user-images.githubusercontent.com/1862756/133346407-20a41eff-4fc0-4013-8a62-d0f58684e22c.png)

With Fix:
![image](https://user-images.githubusercontent.com/1862756/133346432-68e15060-d61f-4c43-89fc-2136153dbd62.png)